src/components/AppliedRoute/index.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import { MemoryRouter } from 'react-router'
+import { Provider } from 'react-redux';
+import Home from '../../screens/Home';
+import Navigation from '../Navigation';
+import store from '../../store';
+import AppliedRoute from './index';
+
+describe("test render AppliedRoute", () => {
```

Comment:
`test` keyword is the most unnecessary in your test 😄 I see it everywhere. We're already in test file, everything we do here is for test, so no need that keyword anymore.
The test structure should be:
```
describe(<Object you wanna test>)
...test(<scenario 1>)
...test(<scenario 2>)
...test(<scenario 3>)
```

Reply:
Thank you for your code review, i have changed the test name as you suggest in the new commit also the indentation of all files